backends/rapidpro/msg.go

```diff
@@ -374,7 +374,7 @@ type DBMsg struct {
 	Visibility_   MsgVisibility          `json:"visibility"      db:"visibility"`
 	HighPriority_ null.Bool              `json:"high_priority"   db:"high_priority"`
 	URN_          urns.URN               `json:"urn"`
-	URNAuth_      string                 `json:"urn_auth"`
+	URNAuth_      string                 `json:"contact_urn_auth"`
```

Comment:
the json key should be "urn_auth"

check https://github.com/nyaruka/rapidpro/blob/master/temba/channels/courier.py#L69

Reply:
@norkans7 Oh, you already fixed it on RapidPro, let me change back